### DISCUSSION TITLE
Project image preview

### DISCUSSION THREAD

**Discussion by bogdanbacila** [hugo-themes/toha#535] [Q&A] [answered]
Hi, 

I've just come across this theme and I rather like it, it suits well for both personal and academic purposes. I do have a question and I'm not sure if this is possible or not. Is it possible to have an image preview for each project? So I know the logo in the left corner, but what I'm asking about is something similar to the image displayed in the posts sections? (I think it is called a hero?). Or some other way in which to link a project to a post, so the same structure like the projects section (with the filter buttons) but with posts instead?

Best, 
Bogdan

**Comment by hossainemruz** [comment, marked as answer]
Hi, Bogdan. Currently, it is not possible to have a hero image for project.

**Reply by SrAdriQ** [reply]
Hello, do you have any plans to add a hero image for projects sections? Or made a section like an recent posts but made it editable?

**Reply by hossainemruz** [reply]
Currently, I don't have any plan for adding hero image in project card. You can copy the template in  `layouts/partials/sections` folder of your repository and modify it according to your need.
